src/TimeRender.cpp

```diff
@@ -0,0 +1,62 @@
+#include "TimeRender.h"
+#include <iostream>
+
+TimeRender::TimeRender(String text, U32 font_size, String type)
+    : text(text), font_size(font_size), type(type)
+{
+	font = TTF_OpenFont("data/fonts/digital-7.ttf", font_size);
+	if (!font)
+	{
+		spdlog::warn("Error loading font: {}", TTF_GetError());
+	}
+}
+
+void TimeRender::init()
+{
+	if (TTF_Init() < 0)
+	{
+		spdlog::error("Error intializing SDL_ttf: {}", TTF_GetError());
+	}
+}
+
+void TimeRender::deinit()
+{
+	TTF_Quit();
+}
+
+auto start = std::chrono::high_resolution_clock::now(); // get current time
+int elapsed_time_ms = 0;
+
+void TimeRenderControlSystem::on_tick()
+{
+	for (auto &&[entity, time_render, position, sprite] : access_storage().each())
+	{
+		sprite.depth = -50;
+		if (!time_render.font)
+		{
+			continue;
+		}
+		if(time_render.type == "time"){
```

Comment:
```suggestion
		if (time_render.type == "time")
		{
```

Also the whole block should be indented.